website/content/en/docs/Getting Started/latest/customization.md

```diff
@@ -53,15 +54,15 @@ spec:
     - name: kubernetes
       version: 1.18.3
     - name: workflow-job
-      version: "2.34"
```

Comment:
The quotes are required here. Without them, 2.34 will be interpreted as a number and operator won't be able to parse the CR.